src/ripple/app/main/Tuning.h

```diff
@@ -25,9 +25,10 @@ namespace ripple {
 enum
 {
      fullBelowTargetSize = 524288
-    ,fullBelowExpirationSeconds = 600
 };
 
+constexpr std::chrono::seconds fullBelowExpiration = std::chrono::minutes{10};
+
```

Comment:
Suggestion: convert lonely `fullBelowTargetSize` enum to `constexpr std::size_t`